flight_declaration_operations/serializers.py

```diff
@@ -39,8 +39,8 @@ class FlightDeclarationRequestSerializer(serializers.Serializer):
     originating_party = serializers.CharField(
         required=False, default="No Flight Information"
     )
-    start_datetime = serializers.DateTimeField()
-    end_datetime = serializers.DateTimeField()
+    start_datetime = serializers.DateTimeField(required=False, default=None)

```

Comment:
Sending flight_time is optional

Reply:
Do we need to have an optional end_datetime as well?

Reply:
`end_datetime` is also optional :)

Reply:
Oh yeah there it is :-D